### PR TITLE
Update PROW config feature tests triggers

### PR DESCRIPTION
**Pull request by smoshiur1237**
This will add and update the triggers which will be ran by PROW in PRs

### CODE REVIEW

File: prow/manifests/overlays/metal3/config.yaml [hunk 1]
```diff
@@ -731,50 +731,146 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
-  # name: {job_prefix}-{image_os}-e2e-feature-test-{capm3_target_branch}
-  - name: metal3-centos-e2e-feature-test-main
+  # name: {job_prefix}-{image_os}-e2e-feature-test-{capm3_target_job}
```
Comment (mboukhalfa) [inline prow/manifests/overlays/metal3/config.yaml:734]:
```suggestion
  # name: {job_prefix}-{image_os}-e2e-feature-test-{capm3_target_branch}-{capm3_target_job}
```

Comment (smoshiur1237) [reply prow/manifests/overlays/metal3/config.yaml:734]: It is not necessary, as `capm3_target_job` values also pointed as `main-pivoting` , `main-remediation` ...... So branch name is covered here.

**Review by Rozzii — COMMENTED** [review]
/lgtm
Let's hope nothing breaks.

**Review by Sunnatillo — APPROVED** [review]
/approve

I do not have a strong opinion on the naming format mention.

**Pull request merged**